app/src/components/v-emoji-picker.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<v-button class="emoji-button" x-small secondary icon @click="emojiPicker.togglePicker($event.target as HTMLElement)">
+		<v-icon name="insert_emoticon" />
+	</v-button>
+</template>
+
+<script setup lang="ts">
+import { EmojiButton } from '@joeattardi/emoji-button';
+import { onUnmounted } from 'vue';
+
+const emojiPicker = new EmojiButton({
+	theme: 'auto',
+	zIndex: 10000,
```

Comment:
Is this going to fly when using it with drawers/dialogs?

Reply:
What do you mean by fly? It should be in front of drawers and dialogs.